contentcuration/contentcuration/frontend/channelEdit/vuex/assessmentItem/mutations.js

```diff
@@ -2,25 +2,49 @@ import Vue from 'vue';
 import { mergeMapItem } from 'shared/vuex/utils';
 
 export function ADD_ASSESSMENTITEM(state, assessmentItem) {
```

Comment:
What I have done in other places is just use the `ADD` mutation for updates too.

Reply:
I did this on purpose just to document a bit that it's really intended (when I saw first time that we use the same actions and mutations for both adding and updating it took me a while to figure out dependency on how we treat our data in terms of IndexedDB) and also to have a place prepared in case we need to separate logic for adding/updating without the need to update calls in all places where we use those mutation because I needed to do exactly the same thing for for add/update actions when I needed to use a bit different logic there for these two operations.

In general, I would like to make clear from the code itself that we are trying to add something new or update something existing.

Though might be personal opinion. Makes sense to discuss with others these conventions in general?